ynput/ayon-cinema4d
# add AYON context to the AYON menu

## Changelog Description
resolve #11 

![Animation_47](https://github.com/user-attachments/assets/85026544-facc-4428-8527-09235b18dbac)


## Testing notes:

1. Launch c4d via AYON Launcher
2. Check if AYON context is shown properly.
